lib/builder/generator.js

```diff
@@ -223,7 +223,32 @@ export default class Generator {
       return false
     }
 
-    if (this.options.generate.minify) {
+    // Legacy workaround for obsolete options.generate.minify
+
+    if (typeof this.options.build.minify === 'undefined') {
+      this.options.build.minify = {
```

Comment:
I think this can be default value of build.minify directly.

Reply:
The problem is that we'd possibly introduce a breaking change because this value is not equal to `minify: true` or the resulting default values (which was the value for SSR/SPA layout minification).

Reply:
I think there actually no difference to users because they don’t need change anything and also wont influence their project features, it’s just some specific options of minifier, let’s ask @pi0 and @Atinux opinion.

Reply:
In the past things like script injection broke because of minification (where closing tags were removed), so I can't judge which problems the changed setting might cause. :see_no_evil: 

But it shouldn't be up to me :relaxed:

Reply:
Yep, but current default options are more conservative than true, right? It should only fix sth instead of breaking 😅